t/t1011-kvs-checkpoint-period.t

```diff
@@ -0,0 +1,165 @@
+#!/bin/sh
+#
+
+test_description='Test kvs module checkpoint-period config.'
+
+. `dirname $0`/kvs/kvs-helper.sh
+
+. `dirname $0`/sharness.sh
+
+RPC=${FLUX_BUILD_DIR}/t/request/rpc
+
+skip_all_unless_have jq
+
+export FLUX_CONF_DIR=$(pwd)
+SIZE=4
+test_under_flux ${SIZE} minimal
+
+kvs_checkpoint_get() {
+        jq -j -c -n  "{key:\"$1\"}" \
+            | $RPC kvs-checkpoint.get \
+            | jq -r .value.rootref
+}
+
+# arg1 - old ref
+# arg2 - timeout (seconds)
+kvs_checkpoint_changed() {
+        old_ref=$1
+        local i=0
+        local iters=$(($2 * 10))
+        while [ $i -lt ${iters} ]
+        do
+            ref=$(kvs_checkpoint_get kvs-primary)
+            if [ "${old_ref}" != "${ref}" ]
+            then
+                return 0
+            fi
+            sleep 0.1
+            i=$((i + 1))
+        done
+        return 1
+}
+
+test_expect_success 'configure bad checkpoint-period timer in kvs' '
+        cat >kvs.toml <<EOF &&
+[kvs]
+checkpoint-period = "1Z"
+EOF
```

Comment:
Suggestion: use `<<-EOF` and the hereis content can be indented wtih tab characters.